cypress/fixtures/adminLogin.json

```diff
@@ -1,7 +1,7 @@
 {
   "email": "purusotam405@gmail.com",
```

Comment:
In real, we have faker(ngfalso is the one we use currently) to use for all the fake data. You wont need to do like this.